src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -32,7 +32,7 @@ Microsoft.CodeAnalysis.Operations.BasicBlock.BranchFlags.StructuredExceptionHand
 Microsoft.CodeAnalysis.Operations.BasicBlock.BranchFlags.Throw = 32 -> Microsoft.CodeAnalysis.Operations.BasicBlock.BranchFlags
 Microsoft.CodeAnalysis.Operations.BasicBlock.Conditional.get -> (Microsoft.CodeAnalysis.IOperation Condition, bool JumpIfTrue, Microsoft.CodeAnalysis.Operations.BasicBlock.Branch Branch)
 Microsoft.CodeAnalysis.Operations.BasicBlock.Kind.get -> Microsoft.CodeAnalysis.Operations.BasicBlockKind
-Microsoft.CodeAnalysis.Operations.BasicBlock.Next.get -> Microsoft.CodeAnalysis.Operations.BasicBlock.Branch
+Microsoft.CodeAnalysis.Operations.BasicBlock.Next.get -> (Microsoft.CodeAnalysis.IOperation ReturnValue, Microsoft.CodeAnalysis.Operations.BasicBlock.Branch Branch)
```

Comment:
>Next [](start = 45, length = 4)

You decided not to put it inside the Branch object?

Reply:
> You decided not to put it inside the Branch object?

Yes, at least for now. I think we can make this change later, if we decide this is the right shape for the API. Then, perhaps, we should move condition into the Branch as well. For now I keep them both outside. 


---
In reply to: [171911265](https://github.com/dotnet/roslyn/pull/25190#discussion_r171911265) [](ancestors = 171911265)